server/controllers/documents.js

```diff
@@ -0,0 +1,204 @@
+import models from '../models';
+import jwt from 'jsonwebtoken';
+import config from '../config/config';
+
+const Documents = models.Documents;
+const Users = models.Users;
+
+const checkDocOwner = (docId, queryId) => {
+  if (parseInt(docId, 10) === parseInt(queryId, 10)) return true;
+};
+
+const checkUserRole = (roleId) => {
+  if (roleId === 1) return true;
+}; 
+
+const documentController = {
+  create(req, res) {
+    const userData = req.body;
+    const loggedInUser = req.user;
+    if( req.body.title === '' || req.body.content === '' 
```

Comment:
I feel it makes for good user experience, if the user receives an bad request message any time any of the require fields is empty.